numba/cuda/tests/cudapy/test_cooperative_groups.py

```diff
@@ -130,10 +131,10 @@ def test_max_cooperative_grid_blocks(self):
         # whilst keeping the total number of threads constant doesn't change
         # the maximum to validate some of the logic.
         c_sequential_rows = cuda.jit(void(int32[:,::1]))(sequential_rows)
-        defn = c_sequential_rows.definition
-        blocks1d = defn.max_cooperative_grid_blocks(256)
-        blocks2d = defn.max_cooperative_grid_blocks((16, 16))
-        blocks3d = defn.max_cooperative_grid_blocks((16, 4, 4))
+        overload = c_sequential_rows.overloads[(int32[:,::1],)]
```

Comment:
factor out `sig = (int32[:, ::1],)` and reuse above and here?

Reply:
Did you wish to do this, I can't seem to find it in the diff? I don't think it's necessarily essential.

Reply:
I've done this now, I missed it before.

Reply:
Thanks